lib/opal/nodes/top.rb

```diff
@@ -108,6 +172,66 @@ def compile_end_construct
       def version_comment
         "/* Generated by Opal #{Opal::VERSION} */"
       end
+
+
+      def self.module_name(filenamepath, original_filepath = nil, original_filename = nil)
+        original_filename = filenamepath unless original_filename
+        original_filepath = Pathname.new(filenamepath).expand_path unless original_filepath
+        o_s = original_filepath.to_s
+        path, _ = Pathname.new(filenamepath).expand_path.split
+        if Opal.paths.include?(path.expand_path.to_s)
+          e = if o_s.end_with?('.js.rb')
+                -7
+              elsif o_s.end_with?('.rb') || o_s.end_with?('.js')
+                -4
+              else
+                -1
+              end
+          return o_s[(path.expand_path.to_s.size+1)..e]
+        end
+        if path.root?
+          pwd = Dir.pwd
+          if o_s.start_with?(pwd)
+            e = if o_s.end_with?('.js.rb')
+                  -7
+                elsif o_s.end_with?('.rb') || o_s.end_with?('.js')
+                  -4
+                else
+                  -1
+                end
+            return o_s[(pwd.size+1)..e]
+          else
+            return o_s
+          end
+        end
+        module_name(path, original_filepath, original_filename)
+      end
+
+      private
+
+      def import_name(m_name)
+        m_name.gsub('.','o_').gsub('-','_').gsub('/','_').gsub('@','_at_') + rand(36**8).to_s(36)
```

Comment:
Please, add a space after comma. I think `rake rubocop` would give you the same suggestion.